### PR TITLE
js: Add AccountInfo to JetStreamManagement

### PR DESCRIPTION
**Pull request by wallyqs**
Signed-off-by: Waldemar Quevedo <wally@synadia.com>

### CODE REVIEW

File: jsm.go [hunk 1]
```diff
@@ -102,22 +105,45 @@ type apiPagedRequest struct {
...
+
+// AccountInfo retrieves info about the JetStream usage from the current account.
+func (js *js) AccountInfo() (*AccountInfo, error) {
+	resp, err := js.nc.Request(js.apiSubj(apiAccountInfo), nil, js.wait)
+	if err != nil {
+		if err == ErrNoResponders {
+			err = ErrJetStreamNotEnabled
+		}
+		return nil, err
+	}
```
Comment (variadico) [inline jsm.go:137]: Should we overwrite `ErrNoResponders` with `ErrJetStreamNotEnabled` here or should we do that in the `JetStream` construction function above?

https://github.com/nats-io/nats.go/pull/639/files#diff-176a2342c55218b70f165bed6a7731dc83d901f9513382f3c98bbd3d08ce0678R134?

I'm thinking about the case where someone calls `AccountInfo` after the constructor function. Not sure which would be more informative. Just a thought.

**Review by variadico — APPROVED** [review]

File: jsm.go (hunk 1 above)
Comment (wallyqs) [reply jsm.go:137]: Good point, will add it back to be done when calling `JetStream()` instead.

Comment (wallyqs) [reply jsm.go:137]: done

File: jsm.go [hunk 2]
```diff
@@ -102,22 +105,45 @@ type apiPagedRequest struct {
...
 }
 
 type accountInfoResponse struct {
 	apiResponse
-	accountStats
+	AccountInfo
+}
+
+// AccountInfo retrieves info about the JetStream usage from the current account.
+func (js *js) AccountInfo() (*AccountInfo, error) {
```
Comment (wallyqs) [inline jsm.go:130]: btw the type that represents this in the server is called `JetStreamAccountStats`
https://github.com/nats-io/nats-server/blob/00db4ba7a1c7520641805ca14e6aae608d37b1d6/server/jetstream.go#L52

**Review by derekcollison — APPROVED** [review]
LGTM

**Pull request merged**